book/modules/appendix2-exercises.tex

```diff
@@ -1,9 +1,80 @@
-
 \begin{exercises}
-
 	\begin{problist}
-		\prob XXX Finish
-
-		XXX Include exercise that asks for an augmented matrix with a row of zeros but where there isn't a free variable
+		\prob Find the complete solution for the following systems.
+		\begin{enumerate}
+			\item $\systeme{x+y-2z=-5,-4x+y+5z=3}$
+			\item $\systeme{3x-2y=-4,x+y+3z=3,-4x+y-3z=1}$
+			\item $\systeme{x-y+2z=-1,2x+y+4z=1,3x-4y+3z=-2}$
+			\item $\systeme{2x+z=8,x+y+z=4,x+3y+2z=4,3x+2y+4z=9}$
+			\item $\systeme[xyzw]{4x+6y+3z-10w=6,5x+2y+z-7w=2,-6x+2y+z+4w=2}$
+			\item $\systeme[xyzw]{2x+2y+z=-1,y-4z+2w=3,x-y-3z-4w=5}$
+		\end{enumerate}
+		\begin{solution}
+			\begin{enumerate}
+				\item 
+				\[
+					\Rref=
+					\begin{bmatrix}[ccc|c]
+					1 & 0 & -7/5 & -8/5\\
+					0 & 1 & -3/5 & -17/5
+					\end{bmatrix}
+				\]
+				\item 
+				\[
+					\Rref=
+					\begin{bmatrix}[ccc|c]
+					1 & 0 & 6/5 & 2/5\\
+					0 & 1 & 9/5 & 13/5\\
+					0 & 0 & 0 & 0
+					\end{bmatrix}
+				\]
+				\item 
+				\[
+					\Rref=
+					\begin{bmatrix}[ccc|c]
+					1 & 0 & 0 & 4/3\\
+					0 & 1 & 0 & 1\\
+					0 & 0 & 1 & -2/3
+					\end{bmatrix}
+				\]
+				\item 
+				\[
+				\Rref=
+				\begin{bmatrix}[ccc|c]
+					1 & 0 & 0 & 5\\
+					0 & 1 & 0 & 1\\
+					0 & 0 & 1 & -2\\
+					0 & 0 & 0 & 0
+				\end{bmatrix}
+				\]
+				\item 
+				\[
+					\Rref=
+					\begin{bmatrix}[cccc|c]
+					1 & 0 & 0 & -1 & 0\\
+					0 & 1 & 1/2 & -1 & 1\\
+					0 & 0 & 0 & 0 & 0
+					\end{bmatrix}
+				\]
+				\item 
+				\[
+					\Rref=
+					\begin{bmatrix}[cccc|c]
+					1 & 0 & 0 & -2 & 1\\
+					0 & 1 & 0 & 2 & -1\\
+					0 & 0 & 1 & 0 & -1
+					\end{bmatrix}
+				\]
```

Comment:
These look nice. Move (e) and (f) to the start. We don't want to start out with fractions in the answer.

Reply:
There is a `1/2` in the rref of (e). Is it okay to have this one fraction in (e) and (f)?

Reply:
Yeah. That is fine.